installInstructions/dockerInstructions.rst

```diff
@@ -0,0 +1,73 @@
+.. _dockerInstructions:
+
+Using Docker
+============
+
+The MACH framework is packaged within Docker images that can be used to run the software without installing it natively on your machine.
+If you are using MACH for the first time, we encourage you to try Docker to avoid installation issues or other inconveniences caused by natively installing the tools.
+
+This guide assumes you are have Docker installed and running on your machine.
+If you need to install Docker, you can follow the Docker guide for `Installing the Docker Engine <https://docs.docker.com/engine/install/>`_.
+The commands used in this guide may differ depending on your operating system, so refer to the Docker documentation for more details for specific use cases.
+
+Pull MDO Lab Docker Image
+-------------------------
+
+Pull one of the MDO Lab Docker images.
+The available images are listed in the table below:
+
+======================== ================
+Tag                      Operating System
+======================== ================
+c7-intel-impi-latest     CentOS 7
+u20-gcc-ompi-latest      Ubuntu 20.04
+u20-gcc-ompi-stable      Ubuntu 20.04
+tacc-u18-gcc-impi-stable Ubuntu 18.04
+======================== ================
+
+To pull an image, use the ``docker pull`` command:
+
+.. code-block:: bash
+
+    $ docker pull mdolab/public:<TAG>
+
+
+Check that the Docker image is pulled successfully by running:
+
+.. code-block:: bash
+
+    $ docker image ls
+
+You should see the image you just pulled.
+
+Initialize Docker Container
+---------------------------
+Navigate to the directory containing the case you would like to run and initialize the Docker image you downloaded into a container, running interactively:
+
+.. code-block:: bash
+
+    $ docker run -it --name <NAME> --mount "type=bind,src=<HOST_DIR>,target=<MOUNT_DIR>" <IMAGE> /bin/bash
```

Comment:
Do we want to tell people to put `$(pwd)` instead of `<HOST_DIR>`?

Reply:
I left this as `<HOST_DIR>` to make it more general and given our previous conversation about how it is more reliable than `$(pwd)`. But, either is fine by me.

Reply:
Yeah I'm fine with the PR as is actually